data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+ARTIFACTS="${1:-/tmp/artifacts}"
+mkdir -p "${ARTIFACTS}"
+
+echo "Gathering bootstrap journals ..."
```

Comment:
Probably write these to stderr, if we plan on using stdout to stream a tarball back to the install host?

Reply:
If we find that managing stdout is found to be too risky we can revert to using scp I guess but it'd be nice if this were a one liner that resulted in a tarball on local host.

Reply:
> ... but it'd be nice if this were a one liner that resulted in a tarball on local host.

Hey, you can `ssh ... && scp ...` all on one line ;).